models/linkedin.yml

```diff
@@ -389,9 +389,10 @@ models:
           CANCELED - Creative is permanently canceled.
 
       - name: click_uri
-        description: |
-          Required when creative type is TEXT_AD. Maximum character limit of 500 after URL encoding all special characters. 
-          Limit does not include the "https://" prefix.
+        description: The URL where the member should be redirected to, on clicking the text ad or spotlight ad.
+
+      - name: click_uri_type
+        description: Will either be `text_ad`, `spotlight`, or `null` to indicate the origin of the click_uri.
```

Comment:
Same question as in the source


> I feel like null can mean the click_uri either originates from the old click_uri field or just another click uri ad type. Do you feel it's fine that this scenario isn't differentiated?

Reply:
Never mind, thanks for answering this in the source!

Reply:
Thanks @fivetran-reneeli from the source discussion, this should be fine as is.